tests/TestCase/Controller/ControllerTest.php

```diff
@@ -258,6 +264,27 @@ public function tearDown()
     {
         parent::tearDown();
         Plugin::unload();
+        error_reporting(self::$error_level);
+    }
+
+    /**
+     * setUpBeforeClass
+     *
+     * @return void
+     */
+    public static function setUpBeforeClass()
+    {
```

Comment:
I'm not sure needs `parent::setUpBeforeClass();` here or no
because we no never use this in cakephp core